src/main/java/org/codeforamerica/messaging/models/Template.java

```diff
@@ -23,10 +24,21 @@ public class Template {
     @Column(unique=true)
     @ToString.Include
     String name;
-    @OneToMany(mappedBy = "template", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
-    List<TemplateVariant> templateVariants;
+    @ToString.Include
+    @Singular
+    @OneToMany(mappedBy = "template", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    List<TemplateVariant> templateVariants = new LinkedList<>();
     @CreationTimestamp
     private OffsetDateTime creationTimestamp;
     @UpdateTimestamp
     private OffsetDateTime updateTimestamp;
+
+    public TemplateVariant addTemplateVariant(TemplateVariant.TemplateVariantBuilder templateVariantBuilder) {
+        if (templateVariants.isEmpty()) {
```

Comment:
Why is this needed? We should be able to add a variant to an empty list.

Reply:
The problem is caused by `Lombok`. We need to add a `@Builder.Default` annotation to the `templateVariants` field to allow it to be initialized to the empty collection.